apps/spruce/src/analytics/projectHealth/useProjectHealthAnalytics.ts

```diff
@@ -6,50 +6,46 @@ import {
 
 type pageType = "Commit chart" | "Task history" | "Variant history";
 type Action =
+  | { name: "Changed page"; direction: "previous" | "next" } // "Commit chart"
```

Comment:
What do these comments represent? They seem to have fairly specific formatting

Reply:
They represent which page they belong to. Since we use the same analytics file for all 3 pages, commit, variant history, and task history I added the comments to help easily distinguish them in the future.